coala_quickstart/green_mode/green_mode_core.py

```diff
@@ -50,8 +50,7 @@ def green_mode(project_dir: str, ignore_globs, bears, bear_settings_obj,
         The maximum number of values to run the bear again and again for
         a optional setting.
     """
-    from coala_quickstart.green_mode.filename_operations import (
-        check_filename_prefix_postfix)
+
```

Comment:
Unnecessary newline, please remove this

Reply:
The build is failing due to a different reason, most probably.